packages/terra-site/webpack.config.js

```diff
@@ -6,6 +6,7 @@ const path = require('path');
 const autoprefixer = require('autoprefixer');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const I18nAggregatorPlugin = require('./node_modules/terra-i18n/lib/I18nAggregatorPlugin');
```

Comment:
I would also recommend updating the syntax to
```
const I18nAggregatorPlugin = require('terra-i18n/lib/I18nAggregatorPlugin');
```
in case the node_modules directory is not relative